blobfuse2-release.yaml

```diff
@@ -527,3 +532,57 @@ stages:
                   changeLogType: 'commitBased'
                   isDraft: false
                   assetUploadMode: replace
+  - stage: UpdateLatestVersion
+    dependsOn: ReleaseArtifact
+    condition: succeeded('ReleaseArtifact')
+    jobs:
+      - job: UpdateVersion
+        pool:
+          vmImage: 'ubuntu-18.04'
+        variables:
+          - group: NightlyBlobFuse
+        
+        steps:
+          - checkout: none
+
+          - ${{ if eq(parameters.update_version, true) }}:
+              - script: |
+                  sudo apt-get install python3 -y
+                  python3 --version
+                displayName: 'Installing Python'
+
+              # download artifacts that need to be published
+              - task: DownloadBuildArtifacts@0
+                displayName: 'Download Build Artifacts'
+                inputs:
+                  artifactName: 'blobfuse2'
+                  downloadPath: $(Build.ArtifactStagingDirectory)
+              
+              # install blobfuse2
+              - script: |
+                  cd $(Build.ArtifactStagingDirectory)/blobfuse2
+                  ls | grep ubuntu-18.04
+                  sudo apt-get install ./`ls | grep ubuntu-18.04` -y
+                  blobfuse2 version
+                displayName: 'Installing blobfuse2'
+
+              # Add '-preview' to pre-release versions. For example, 2.0.0-preview.1
+              # Also considering that the tag and version are same
+              - script: |
+                  python3 -c "import json; import requests; import sys;
```

Comment:
We initially had python in some of our pipelines and sometimes we had dependency issues on the VMs. This seems safe enough since its a MS hosted VM however - I don't like the idea of having the python script be part of the yaml. Can we instead make this a separate pythons cript file (or preferably written in GO - refer to config-gen.go)

Reply:
The reason why I kept this inline is because we don't have access to the code in this step. So, if we were to maintain this in a separate script, we will have to clone the repo and then run the script. Let me know what are your thoughts on this.